liquidator/index.js

```diff
@@ -84,7 +84,8 @@ async function run(address, pollingDelay, priceFeedConfig, liquidatorConfig) {
     }
 
     while (true) {
-      await liquidator.queryAndLiquidate();
+      const currentSyntheticBalance = await syntheticToken.balanceOf(accounts[0]);
```

Comment:
I elected to implement it this way because I didn't want to pass additional params into the Liquidator bot so that it could access the `syntheticToken` contract to call `balanceOf()`, but I think this is nicely modular